packages/perseus-editor/src/widgets/interactive-graph-editor/components/interactive-graph-sr-tree.tsx

```diff
@@ -0,0 +1,150 @@
+import Pill from "@khanacademy/wonder-blocks-pill";
+import * as React from "react";
+
+import Heading from "../../../components/heading";
+
+type Attribute = {
+    name: string;
+    value: string;
+};
+
+type AttributeMap = {
+    role: string;
+    attributes: Array<Attribute>;
+};
+
+// Exported for testing
+export function fetchAriaLabels(container?: Element): AttributeMap[] {
+    const elementArias: Array<AttributeMap> = [];
+
+    if (!container) {
+        return elementArias;
+    }
+
+    // Traverse through all the children of the container and
+    // collect the aria-labels and aria-descriptions.
+    container.querySelectorAll("*").forEach((element) => {
+        // List to return with all the aria-labels and aria-descriptions.
+        const elementAttributes: Array<Attribute> = [];
+
+        const attributes = element.attributes;
+        for (const attribute of attributes) {
```

Comment:
Why use a loop to get the attributes here? It seems like it might be simpler to look up the specific attributes we care about, e.g.

```ts
const ariaLabel = element.getAttribute("aria-label");
```

Then we'd have more direct control over the order in which the attributes are displayed.

I don't think an element can have multiple attributes with the same name, so this preserves behavior.

Reply:
Ah!! I'm so silly.

I tried that, but then my holiday brain thought that `aria-describedby` can be added multiple times, rather than it being one attribute that's space-separated strings 🤦🏽‍♀️